server.js

```diff
@@ -6,17 +6,49 @@ const io = require('socket.io')(http);
 const port = 5000;
 const host = 'localhost';
 
+var names = [];
+var words = [];
+var players = [];
+
+//TODO: User this object to print final point, and give points do the righ user
+var user = {name: null, word: null, points: 0}
+
 app.use(express.static(__dirname + '/public'));
+app.set('views', __dirname + '/public/views');
+app.engine('html', require('ejs').renderFile);
+app.set('view engine', 'html');
 
-//History with drawed lines to render on new client connection
-const drawedList = [];
```

Comment:
Why you deleted this list?
That is needed to other users can see on start new game or refresh de page
Need to be implemented